aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -171,6 +214,7 @@ cublasHandle_t getCurrentCUDABlasHandle() {
     workspace_it = cublas_handle_stream_to_workspace().insert(workspace_it, {key, getNewWorkspace()});
   }
   TORCH_CUDABLAS_CHECK(cublasSetWorkspace(handle, workspace_it->second.get(), getChosenWorkspaceSize()));
+#if !defined(USE_ROCM)
```

Comment:
\nit `#if defined(USE_CUDA)` would be cleaner looking, mostly so we don't have to think about negation.